### PR TITLE
Address documentation issues in 4615

### PR DESCRIPTION
**Pull request by rglazer2**
Addresses #4615

### CODE REVIEW

**Comment by Myoldmopar** [comment]
@rglazer2 @JasonGlazer I'm going to close this PR for right now so that @nrel-bot doesn't start complaining about it.  I won't delete the branch, so whenever you get a chance to remedy what I mentioned [here](https://github.com/NREL/EnergyPlus/pull/4990#issuecomment-116803148), just push those changes up and re-open the PR and we'll then work to get it merged.  Thanks.

File: idd/V8-2-0-Energy+.idd [hunk 1]
```diff
@@ -772,7 +772,8 @@ ConvergenceLimits,
        \minimum 2
 
 ProgramControl,
-       \memo used to support various efforts in time reduction for simulation including threading
+       \memo Used to support various efforts in time reduction for simulation including threading
```
Comment (mjwitte) [inline idd/V8-2-0-Energy+.idd:775]: @rglazer2 Please add a period at the end of this line.

Comment (mjwitte) [inline idd/V8-2-0-Energy+.idd:775]: @rglazer2 This is the wrong IDD file to put the changes inn.  Should be idd/Energy+.idd.in

Comment (mjwitte) [inline idd/V8-2-0-Energy+.idd:775]: @rglazer2 The ProgramControl changes are still in the wrong idd file.

**Comment by mjwitte** [comment]
@rglazer2 Getting closer.  For some reason, the diffs are still tripping on these source files.  Please try to reset line endings (again) and push these up.
doc/src/docs/InputOutputReference/01a-InputOutputReference.md
doc/src/docs/InterfaceDeveloper/InterfaceDeveloper.md
doc/src/docs/OutputDetailsAndExamples/OutputDetailsAndExamples.md

**Comment by Myoldmopar** [comment]
Umm, I'm a little concerned about the list of commits above, most notably that in the process here you reverted out some previous work.  From the diff, I think you got everything back in place, but that will still be in the history when we push the green button.  You might want to consider starting a new branch and copy/pasting the files you've changes in.  You'd want to do the following things:
- Stash away the current state of the files you've changed in this branch
- Update your clone with the latest version of develop from NREL/EnergyPlus
- Checkout a new branch based on the develop branch
- Paste your changed files back in to the folder
- Commit and push this new branch up to your fork, and open a new pull request (close this one)

Let me know if you need anything.

**Comment by mjwitte** [comment]
@rglazer2 @Myoldmopar Yeah, looks like the many attempts to get rid of the 10,000 delete/add in the diff has failed.  Because there are changes to some of the same files in the other doc PR, I'm afraid making a new branch and copying the files in as-is will not work.  Probably need to just start over and do some interactive diff/merge after #4990 is merged.  Also, ready to give up on using a forked repo.

**Comment by Myoldmopar** [comment]
@rglazer2 Do you anticipate using this pull request?  Given the conflicts and issues, and the relatively small amount of actual code/text changes, it seems better to just start a new branch.  It's up to you.  But if you decide to do a new one, could you close this pull request unmerged so that our automated robots don't start complaining about it going out of date.  

Let me know if you have any questions.

**Comment by rglazer2** [comment]
@Myoldmopar @mjwitte Since I can't seem to fix the line endings issue for this issue, I will close this pull request and copy/redo the changes in a new branch.

**Pull request closed without merging**